src/game/entities/Serializable.ts

```diff
@@ -4,10 +4,18 @@ import {
   SerializableConstructor,
   UpdateAction,
 } from '../../types/gameTypes';
-import { getEntityFromType, toFixedIfNecessary } from '../utils';
+import {
+  applyBasicUpdateTolerance,
+  getEntityFromType,
+  toFixedIfNecessary,
+} from '../utils';
 import GameObject from './GameObject';
 
 export default abstract class Serializable extends GameObject {
+  //#region Debug
+  maxXTolerance = 0;
```

Comment:
This field is the max difference between the client x and the received server X (on a basic update) on any given frame.

Reply:
then the name is misleading
```suggestion
  maxDeviationRegistered = 0;
```